### PR TITLE
kubernetes-sigs/k8s-container-image-promoter: add prow plugins

### PR DESCRIPTION
**Pull request by cpanato**
Add common prow plugins to kubernetes-sigs/k8s-container-image-promoter repo

ref: https://github.com/kubernetes-sigs/k8s-container-image-promoter/issues/266

/cc @justaugustus @nikhita

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -947,6 +947,15 @@ plugins:
   kubernetes-sigs/security-profiles-operator:
   - release-note
 
+  kubernetes-sigs/k8s-container-image-promoter:
+  - blockade
```
Comment (nikhita) [inline config/prow/plugins.yaml:951]: blockade, milestone, milestoneapplier, require-matching-label plugins would also need be specifically configured for the repo.

For example, for milestone_applier, you'll need to tell what milestones to apply:

https://github.com/kubernetes/test-infra/blob/12c654eaf2c5aa900963d63233df08e2fcb91d94/config/prow/plugins.yaml#L327-L335

Comment (cpanato) [reply config/prow/plugins.yaml:951]: ahh, thanks! I think we can drop those, for now, don't think this is a hard requirement for the repo


thanks for your review @nikhita

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

**Pull request merged**